devel/python/python/ert_gui/ertwidgets/models/selectable_list_model.py

```diff
@@ -0,0 +1,46 @@
+from PyQt4.QtCore import QObject, pyqtSignal
+
+
+class SelectableListModel(QObject):
```

Comment:
If this inherits `QAbstractListModel` it can be plugged directly into any of Qt's `ListView`.